### PR TITLE
FLA-681 Added SubmissionSheet link and icon

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

https://justice.gov.bc.ca/jira/browse/FLA-681

- Updated PackageReview to include "Print Submission Sheet" link and icon
  - the icon was obtained from Marie Anik and uploaded to src. @alexjoybc, should we use **react-icons/md** instead?
- Updated PackageReview unit tests and snapshots to include new functionality

Most of the changes in this MR is just generated snapshot changes (when reviewing, you can ignore the *.storyshot and *.snap files)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

yarn test -u (to update snapshots)
yarn test
yarn lint

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by alexjoybc** [comment]
Yes @KDNeufeld we should use the react icon library

File: src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+const downloadSubmissionSheet = (packageId) => {
+  axios
+    .get(`/filingpackage/${packageId}/submissionSheet`, {
+      responseType: "blob",
+    })
+    .then((response) => {
+      const fileData = new Blob([response.data], { type: "application/pdf" });
+      const fileUrl = URL.createObjectURL(fileData);
+
+      FileSaver.saveAs(fileUrl, "SubmissionSheet.pdf");
```
[the remark this answers is not in the document]
Comment (alexjoybc) [reply src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js:23]: I also feel like the backend should drive the filename, so for now we should keep it as is

File: src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js [hunk 2]
```diff
@@ -126,7 +147,22 @@ export default function PackageReview({ page: { header, packageId } }) {
             </div>
           )}
           <br />
-          <h2>Package Details</h2>
+          <div className="row">
+            <h2 className="col-sm-6">Package Details</h2>
+            <div className="col-sm-6 text-right">
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js:152]: I think on mobile we could move the text bellow and not having it stick to the right

image

File: src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js [hunk 3]
```diff
@@ -126,7 +147,22 @@ export default function PackageReview({ page: { header, packageId } }) {
...
+                onKeyDown={() => downloadSubmissionSheet(packageId)}
+                role="button"
+                tabIndex={0}
+                className="file-href"
+                onClick={() => downloadSubmissionSheet(packageId)}
+                data-test-id="uploaded-file"
+              >
+                Print Submission Sheet
+              </span>
+              <BsEyeFill size="24" color="#7F7F7F" className="align-icon" />
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js:163]: nice

**Review by alexjoybc — CHANGES_REQUESTED** [review]
Thanks @KDNeufeld that looks good, I just posted a comment on the mobile view, the text might be better align to the left and the icon on the other side, maybe a quick check with @marieanik

File: src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js (hunk 2 above)
Comment (KDNeufeld) [reply src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js:152]: Excellent suggestion. Done.

File: src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js:23]: Sounds good. We can bring it up in salad.

**Review by alexjoybc — APPROVED** [review]
Looks good thanks @KDNeufeld

**Pull request merged**